dashboard/db/migrate/20200429234529_drop_contact_rollups_comparison.rb

```diff
@@ -0,0 +1,9 @@
+class DropContactRollupsComparison < ActiveRecord::Migration[5.0]
+  def up
+    drop_table :contact_rollups_comparisons
+  end
+
+  def down
```

Comment:
Following examples for irreversible migrations at https://api.rubyonrails.org/classes/ActiveRecord/Migration.html and https://stackoverflow.com/a/31657066.